.github/integration/tests/20_upload.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+set -e
+# inferred from access_key in testing/s3cmd.conf
+user=test_dummy.org
+
+
+# Function checking that a file was encrypted
+function check_encypted_file() {
```

Comment:
This function is redefined and not sure if it will be duplicated more in the future. Maybe it is better to define it in a separate function call e.g. `common_func.sh` and then source it from there.

Reply:
Fixed